lib/cocoapods/command/repo/push.rb

```diff
@@ -31,6 +31,7 @@ def self.options
             ['--no-private', 'Lint includes checks that apply only to public repos'],
             ['--skip-import-validation', 'Lint skips validating that the pod can be imported'],
             ['--skip-tests', 'Lint skips building and running tests during validation'],
+            ['--test-specs=test-spec1,test-spec2,etc', 'List of test specs to run'],
```

Comment:
should that be available for `push`? I'd suspect we want people to push with all tests ran or none. Maybe we can limit this on lib/spec `lint`?

Reply:
I'll revert this file.